ios/Classes/Codables.swift

```diff
@@ -77,7 +77,7 @@ class PolarDataCodable<T>: Encodable {
 }
 
 typealias PolarHrSample = (
-  hr: UInt8, rrsMs: [Int], rrAvailable: Bool, contactStatus: Bool, contactStatusSupported: Bool
+    hr: UInt8, ppgQuality: UInt8, correctedHr: UInt8, rrsMs: [Int], rrAvailable: Bool, contactStatus: Bool, contactStatusSupported: Bool
```

Comment:
These fields need added to `PolarHrSample` in Dart
It also needs handled in the `encode` function